source/components/JsxParser.tsx

```diff
@@ -162,10 +169,22 @@ export default class JsxParser extends React.Component<TProps> {
 			case '!': return !expression.argument.value
 			}
 			return undefined
+		case 'ArrowFunctionExpression':
+		case 'FunctionExpression':
```

Comment:
Do we need to support `FunctionExpression` at all, if we're only allowing the `() => value` non-body syntax?